apps/fulfillment/vite.config.sw.ts

```diff
@@ -0,0 +1,27 @@
+import tsconfigPaths from 'vite-tsconfig-paths';
+import { defineConfig } from 'vite';
+import { Plugin } from 'vite';
```

Comment:
Hm, it looks like prettier is not working for some reason... can you please check why?
```suggestion
import { defineConfig, Plugin } from 'vite';
```

Reply:
Fixed with prettier plugin update.